src/connectors/okta.py

```diff
@@ -146,19 +146,33 @@ def ingest(table_name, options):
         result = response.json()
 
         for row in result:
-            try:
-                row['users'] = requests.get(
-                    url=row['_links']['users']['href'], headers=headers
-                ).json()
-            except TypeError:
-                log.info(row)
-                raise
+            users = []
+            has_next_page = True
+            url = row['_links']['users']['href']
+            while has_next_page:    
+                try:
+                    page = requests.get(
+                        url=url, headers=headers
+                    )
+                    users += page.json()
+                    if 'rel="next"' in page.headers['link']:
+                        for item in str.split(page.headers['link'], ','):
+                            if 'rel="next"' in item:
+                                url = re.findall(r'(?<=\<)(.*?)(?=\>)', item)[0]
```

Comment:
could you use same code block as above, and maybe make it into a helper function?

        url = ''
        links = requests.utils.parse_header_links(response.headers['Link'])
        for link in links:
            if link['rel'] == 'next':
                url = link['url']

Reply:
Change made as requested